integration-tests/integration_test.go

```diff
@@ -834,6 +843,90 @@ func (s *ProcessListeningOnPortTestSuite) TestProcessListeningOnPort() {
 	assert.True(s.T(), hasOpenAndClose(endpoints9091), "Did not capture open and close events for port 9091")
 }
 
+func (s *SymbolicLinkProcessTestSuite) SetupSuite() {
+
+	s.metrics = map[string]float64{}
+	s.executor = NewExecutor()
+	s.StartContainerStats()
+	s.collector = NewCollectorManager(s.executor, s.T().Name())
+
+	s.collector.Env["COLLECTOR_CONFIG"] = `{"logLevel":"debug","turnOffScrape":false,"scrapeInterval":2}`
+	s.collector.Env["ROX_PROCESSES_LISTENING_ON_PORT"] = "true"
+
+	err := s.collector.Setup()
+	s.Require().NoError(err)
+
+	err = s.collector.Launch()
+	s.Require().NoError(err)
+	time.Sleep(30 * time.Second)
+
+	processImage := qaImage("quay.io/rhacs-eng/qa", "collector-processes-listening-on-ports")
+
+	containerID, err := s.launchContainer("process-ports", "-v", "/tmp:/tmp", processImage)
+
+	s.Require().NoError(err)
+	s.serverContainer = containerShortID(containerID)
+
+	actionFile := "/tmp/action_file_ln.txt"
+
+	_, err = s.collector.executor.Exec("sh", "-c", "rm "+actionFile+" || true")
+
+	_, err = s.execContainer("process-ports", []string{"ln", "-s", "/process-listening-on-ports", "/plop"})
```

Comment:
do we need to use an absolute path for `ln` like we do with bash on 877?

Reply:
Also, can't we perform this step during the container build? If we set the container up with the necessary symlinks we can just change the entrypoint when we start the container and remove a fair bit of additional complexity from this test setup

Reply:
Done